doc/build-macos.md

```diff
@@ -41,10 +39,13 @@ Build Gridcoin
         ./autogen.sh
         ./configure
 	make
+	
+    make clean 	[cleans out previous builds!!!!!! Do this between versions]
```

Comment:
make and make clean need to be indented, and make clean needs to be moved up a step to join it with the autogen/configure block.